### PR TITLE
Add AMDF 2.0 schema and documentation

### PR DESCRIPTION
**Pull request by PascalSchoen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,343 @@
+<<<<<<< HEAD
```
Comment (moelders) [inline README.md]: Please, remove the merge conflict tags, line 1, 4 and 343.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,343 @@
...
 # amdf-documentation
 Advanced Material Data Format Documentation
+=======
+# Advanced Material Data Format
```
Comment (moelders) [inline README.md]: There are several titles, I thing the lines 2 and 3 should be removed.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,343 @@
...
+*Version: 2.0*
```
Comment (moelders) [inline README.md]: Always blank line after a section title.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,343 @@
...
+```c++
+hsvColor desatColor(
+    hsvColor recoloredPixel,
+    hsvColor targetColor)
+{
+    float s_t = targetColor.s;
+    float s_d = 1.0 - pow(1.0 - s_t, 5.0);
+
+    hsvColor c_d;
+	c_d.h = recoloredTexture.h;
```
Comment (moelders) [inline README.md]: Is indentation correct?

**Review by moelders — COMMENTED** [review]
Just some minor issues in the MD file.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,343 @@
+<<<<<<< HEAD
 # amdf-documentation
 Advanced Material Data Format Documentation
+=======
+# Advanced Material Data Format
+*Version: 2.0*
+
+## Content
+
+1. [Description](#description)
```
Comment (RecuencoJones) [inline README.md:7]: Use `1.` for all items, GitHub will render numbers accordingly

File: README.md (hunk 1 above)
Comment (PascalSchoen) [reply README.md]: Oh yeah, comes from an interactive rebasing... I didn't look into the file afterwards.

File: README.md (hunk 4 above)
Comment (PascalSchoen) [reply README.md]: Normally blanks are used, but on those lines it added tabs somehow.

File: README.md (hunk 5 above)
Comment (PascalSchoen) [reply README.md]: Ok, so you mean:
1. \[Description\](\#description)
1. \[Material Types\](\#material-types)
1. \[Color Spaces\](\#color-spaces)
...

**Comment by PascalSchoen** [comment]
Your points should be fixed with the last commit.

**Comment by moelders** [comment]
Yeah, it LGTM.

**Review by moelders — APPROVED** [review]
LGTM

**Pull request merged**